phpunit/fonts-library/class-wp-font-family-test.php

```diff
@@ -0,0 +1,182 @@
+<?php
+/**
+ * Tests for Font Family Utils class
+ *
+ * @package Gutenberg
+ */
+
+class WP_Font_Family_Test extends WP_UnitTestCase {
+
+	const FONT_DATA_1 = array(
+		'name'       => 'Piazzolla',
+		'slug'       => 'piazzolla',
+		'fontFamily' => 'Piazzolla',
+		'fontFace'   => array(
+			array(
+				'fontFamily' => 'Piazzolla',
+				'fontStyle'  => 'italic',
+				'fontWeight' => '400',
+				'src'        => 'http://fonts.gstatic.com/s/piazzolla/v33/N0b72SlTPu5rIkWIZjVgI-TckS03oGpPETyEJ88Rbvi0_TzOzKcQhZqx3gX9BRy5m5M.ttf',
+			),
+		),
+	);
+
+	const FONT_DATA_2 = array(
+		'name'       => 'Arial',
+		'slug'       => 'arial',
+		'fontFamily' => 'Arial',
+	);
+
+	function test_get_data() {
+		$font = new WP_Font_Family( self::FONT_DATA_1 );
+		$this->assertEquals( $font->get_data()['slug'], 'piazzolla' );
```

Comment:
Nitpick:
I think it should be the other way around.
The first argument of the `assertEquals` method should be the expected value, and the second one should be the actual value.
This also applies to other usages of assertEquals in this class.
```suggestion
		$this->assertEquals( 'piazzolla', $font->get_data()['slug'] );
```

Reply:
Good. Done!